src/HashBus.Projector.MostMentioned/MostMentionedProjection.cs

```diff
@@ -20,17 +21,17 @@ public MostMentionedProjection(IRepository<string, IEnumerable<Mention>> mention
             this.mentions = mentions;
         }
 
-        public void Handle(TweetAnalyzed message)
+        public Task Handle(TweetAnalyzed message, IMessageHandlerContext context)
         {
             if (!message.Tweet.UserMentions.Any())
             {
-                return;
+                return Task.FromResult(0);
             }
 
             var trackMentions = this.mentions.GetAsync(message.Tweet.Track).GetAwaiter().GetResult().ToList();
             if (trackMentions.Any(mention => mention.TweetId == message.Tweet.Id))
             {
-                return;
+                return Task.FromResult(0);

```

Comment:
given that we're targeting `4.6.1` we can use `Task.CompletedTask` as return value